cucumber/authenticators_gce/features/authn_gce_hosts.feature

```diff
@@ -0,0 +1,32 @@
+Feature: GCP Authenticator - Different Hosts can authenticate with GCE authenticator
+
+  In this feature we define GCE authenticator in policy, test with different
+  host configurations and perform authentication with Conjur.
+
+  Background:
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-gce
+      body:
+      - !webservice
+
+      - !group apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
```

Comment:
why do you need to be a super user? is this for future tests? (e.g create a host with different annotations)

Reply:
nope.